package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/single-spa-svelte.js",
   "scripts": {
     "build": "rimraf lib && babel src/single-spa-svelte.js --out-file lib/single-spa-svelte.js --source-maps",
```

Comment:
Babel is still being used here - why was the `.babelrc` removed?

Reply:
My mistake, reverting

Reply:
babel is still missing from devDependencies after the latest commits. I'd like to also change the package.json to use `exports` rather than `main`, with `type: "module"`, but can do that in a follow-up pull request if you do not have the time to do so in this one. The module format of the code in the lib folder should be ESM

Reply:
I don't have time to do so at the moment, so have to add it a later point.